package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "habits",
+  "version": "0.1.0",
+  "main": "./main.js",
+  "scripts": {
+    "start": "electron .",
+    "postinstall": "for file in `ls .git-hooks`; do ln -sf ../../.git-hooks/${file} .git/hooks/${file}; done"
+  },
+  "devDependencies": {
+    "electron-prebuilt": "^0.36.8",
+    "eslint": "^2.2.0",
+    "eslint-config-airbnb": "^6.0.2",
+    "eslint-plugin-react": "^4.1.0"
```

Comment:
React plugin is an Airbnb's dependency. They might resolve this in future release or on dev branch.